### PR TITLE
Email/Password authentication

### PR DESCRIPTION
**Pull request by SimonLab**
ref: #247

### CODE REVIEW

File: config/config.exs [hunk 1]
```diff
@@ -49,4 +41,11 @@ config :fields, Fields.AES,
 config :fields, Fields,
   secret_key_base: System.get_env("SECRET_KEY_BASE")
 
+# Google application configuration
+config :elixir_auth_google,
```
Comment (nelsonic) [inline config/config.exs]: seems like we can remove this now? https://github.com/dwyl/elixir-auth-google/pull/24

File: mix.exs [hunk 2]
```diff
@@ -50,7 +50,7 @@ defmodule App.MixProject do
       {:gettext, "~> 0.11"},
       {:jason, "~> 1.0"},
       {:plug_cowboy, "~> 2.0"},
-      {:elixir_auth_google, "~> 0.1.0" },
+      {:elixir_auth_google, "~> 1.0.1" },
```
Comment (nelsonic) [inline mix.exs]: version `1.0.2` has the changes made in PR https://github.com/dwyl/elixir-auth-google/pull/24

File: test/app/ctx_test.exs [hunk 3]
```diff
@@ -79,7 +79,7 @@ defmodule App.CtxTest do
     end
 
     test "list_status/0 returns all status" do
-      status = status_fixture()
+      status_fixture()
```
Comment (nelsonic) [inline test/app/ctx_test.exs]: 👍

**Comment by SimonLab** [comment]
Most updated version for email authentication can be found at #42

**Pull request closed without merging**